src/include/gurt/common.h

```diff
@@ -156,7 +156,7 @@ extern "C" {
 		int _esz = (int)(size);					\
 		int _sz = (int)(size) * (cnt);				\
 		/* Compiler check to ensure type match */		\
-		typeof(newptr) optr = oldptr;				\
+	__typeof__(newptr) optr = oldptr;				\
```

Comment:
I meant to remove the tab or space after the variable setting.

Reply:
sorry